### PR TITLE
API-11962 Supplemental claims PII expunge

### PR DESCRIPTION
**Pull request by ElleeB**
Added PII expunge worker, specs, and settings for Supplemental Claims.

### CODE REVIEW

File: config/settings.yml [hunk 1]
```diff
@@ -234,6 +235,10 @@ modules_appeals_api:
       class: AppealsApi::NoticeOfDisagreementUploadStatusBatch
       description: "Update NoticeOfDisagreement statuses with their Central Mail status"
       every: "60m"
+    appeals_api_sc_cleanup_pii:
```
Comment (cilestin) [inline config/settings.yml:238]: I think you're using an older version of the settings, as this whole `job_schedules` section should have been removed (it was from devops, at least...). This should be added to `sidekiq-schedule.yml` instead.

**Review by cilestin — CHANGES_REQUESTED** [review]

File: modules/appeals_api/app/workers/appeals_api/supplemental_claim_clean_up_week_old_pii.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+
+module AppealsApi
+  class SupplementalClaimCleanUpWeekOldPii
```
[the remark this answers is not in the document]
Comment (ElleeB) [reply modules/appeals_api/app/workers/appeals_api/supplemental_claim_clean_up_week_old_pii.rb:6]: Lol. Great!

File: config/settings.yml (hunk 1 above)
Comment (ElleeB) [reply config/settings.yml:238]: ohhhhhh! ok on it.

Comment (ElleeB) [reply config/settings.yml:238]: Heads Up: I'm still seeing the schedules in settings :/

**Review by cilestin — APPROVED** [review]

**Pull request merged**